homeassistant/components/cover/gogogate2.py

```diff
@@ -92,6 +93,13 @@ def supported_features(self):
         """Flag supported features."""
         return SUPPORT_OPEN | SUPPORT_CLOSE
 
+    @property
+    def state_attributes(self):
+        """Return the state attributes."""
+        data = super().state_attributes
+        data[ATTR_TEMPERATURE] = self._temperature
```

Comment:
This should not be added as an attribute but instead be made into it's own sensor. This means that you will need to greate a gogogate2 component to handle the connection and then setup the cover and sensor platforms.

Reply:
Fair enough. For now, I'll update the PR to get the bug fix in and look at adding the component/sensors later.

For additional data points such as battery_level of the garage tilt sensor - should that be added as an attribute of the cover platform or should it be its own sensor too?